examples/test_examples.py

```diff
@@ -51,6 +53,10 @@ def get_setup_file():
     return args.f
 
 
+def is_cuda_and_apex_avaliable():
+    return torch.cuda.is_available() and is_apex_available()
+
+
```

Comment:
This should also depend from the `USE_CUDA` flag, similarly to the rest of the test suite. Even if we decide to set `USE_CUDA` to `True` by default, setting `USE_CUDA` to `False` should result in the examples not using CUDA imo.

Reply:
I think you can do so by adding the following line:

```py
from transformers.testing_utils import torch_device
```
and updating the statement to
```py
return torch.cuda.is_available() and is_apex_available() and torch_device == "cuda"
```

Reply:
Good catch @LysandreJik